pkg/server/server.go

```diff
@@ -246,16 +246,16 @@ func (s *SSIServer) ManifestAPI(service svcframework.Service) (err error) {
 	applicationsHandlerPath := V1Prefix + ManifestsPrefix + ApplicationsPrefix
 	responsesHandlerPath := V1Prefix + ManifestsPrefix + ResponsesPrefix
 
-	s.Handle(http.MethodPut, manifestHandlerPath, manifestRouter.CreateManifest)
+	s.Handle(http.MethodPut, manifestHandlerPath, manifestRouter.CreateManifest, middleware.Webhook(webhookService, webhook.Manifest, webhook.Create))
 
 	s.Handle(http.MethodGet, manifestHandlerPath, manifestRouter.GetManifests)
 	s.Handle(http.MethodGet, path.Join(manifestHandlerPath, "/:id"), manifestRouter.GetManifest)
-	s.Handle(http.MethodDelete, path.Join(manifestHandlerPath, "/:id"), manifestRouter.DeleteManifest)
+	s.Handle(http.MethodDelete, path.Join(manifestHandlerPath, "/:id"), manifestRouter.DeleteManifest, middleware.Webhook(webhookService, webhook.Manifest, webhook.Delete))
 
-	s.Handle(http.MethodPut, applicationsHandlerPath, manifestRouter.SubmitApplication)
+	s.Handle(http.MethodPut, applicationsHandlerPath, manifestRouter.SubmitApplication, middleware.Webhook(webhookService, webhook.Application, webhook.Create))
```

Comment:
Should the verb here be submit?

Reply:
Yea I was thinking about that, I think its the only thing in the entire system that you 'submit' so submit would only be used for that, and it actually creates a cred app in our system, so thats why I left it as 'create'

Reply:
My preference is still 'submit', but as long as this is properly documented and explained to external developers then 👌